ffi/framebuffer_mxcfb.lua

```diff
@@ -516,10 +516,28 @@ local function refresh_mtk(fb, is_flashing, waveform_mode, x, y, w, h, dither)
     if dither and fb.device:canHWDither() then
         fb.update_data.flags = bor(fb.update_data.flags, C.MTK_EPDC_FLAG_USE_DITHERING_Y4)
     end
+    if fb.swipe_animations then
+        fb.update_data.flags = bor(fb.update_data.flags, C.MTK_EPDC_FLAG_ENABLE_SWIPE)
+        fb.swipe_animations = false
+    end
 
     return mxc_update(fb, C.MXCFB_SEND_UPDATE_MTK, fb.update_data, is_flashing, waveform_mode, x, y, w, h, dither)
 end
 
+-- Expose swipe animations
+function framebuffer:_MTK_ToggleSwipeAnimations(enabled)
```

Comment:
toggle doesn't strike me as quite the right name btw; it seems more of a set. In which case you might also want a get and a convenience actual toggle if desired.